### PR TITLE
Update README.md w.r.t meta collection functionality

### PR DESCRIPTION
**Pull request by rohitthakur2590**
Signed-off-by: Rohit Thakur <rohitthakur2590@outlook.com>

##### SUMMARY

resolves: https://github.com/ansible-collections/ansible.network/issues/8

##### ISSUE TYPE

- Bugfix Pull Request
- Docs Pull Request

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,10 +48,50 @@ collections:
 
 **NOTE**: For Ansible 2.9, you may not see deprecation warnings when you run your playbooks with this collection. Use this documentation to track when a module is deprecated.
 
+### Using ``ansible.network`` meta collection to install supported network content collections.
```
Comment (Qalthos) [inline README.md:51]: Versions aren't pinned so might want to add a note that the latest available will be installed

File: README.md [hunk 2]
```diff
@@ -1,14 +1,27 @@
...
+- openvswitch.openvswitch
+- vyos.vyos
+ ```
+
 
 <!--start requires_ansible-->
 ## Ansible version compatibility
 
-This collection has been tested against following Ansible versions: **>=2.9.10,<2.11**.
+This collection has been tested against following Ansible versions: *****.
```
Comment (Qalthos) [inline README.md:24]: This still has to be at least `>=2.9.10`, right?
```suggestion
This collection has been tested against following Ansible versions: **>=2.9.10**.
```

**Review by Qalthos — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (rohitthakur2590) [reply README.md]: corrected!

File: README.md (hunk 1 above)
Comment (rohitthakur2590) [reply README.md]: Updated!

**Review by trishnaguha — APPROVED** [review]

**Pull request merged**